### ISSUE TITLE
LetsEncrypt not working

### ISSUE THREAD

**Issue by SkabbMask** [raspiblitz/raspiblitz#3048]
Hi!
I'm having some issues with setting up the ip2tor bridge combined with the letsencrypt over duckdns, I have very limited experience with network stuff in general so it's probably just something I'm doing wrong but I've spent so many hours troubleshooting it now that I realized I should probably ask for help.

I can access the btcpay server via the ip provided by ip2tor bridge, and the tor address without any problem, as well as through the duckdns subdomain. When I visit it with http:// I get to an nginx start page saying that the web server is working properly, but when I visit it with https:// I get to chose between the two available ip2tor bridges and sign up to pay (again? It's already marked as active in raspiblitz).

I followed this when setting up (and got no errors in that process): https://coincharge.io/en/raspiblitz-btcpay-server/

But now the btcpay server info screen looks like this (when letsencrypt is active).
![Capture1](https://user-images.githubusercontent.com/9406559/161391268-67fc812d-5e01-495d-97d6-b57968efce06.JPG)

Or does the "yet" indicate that I simply just have to wait?

**Comment by rootzoll** [comment]
> When I visit it with http:// I get to an nginx start page saying that the web server is working properly, but when I visit it with https:// I get to chose between the two available ip2tor bridges and sign up to pay (again? It's already marked as active in raspiblitz).

Not sure what this means .. how are those options "to chose between" are presented to you from the web browser? Can you share more details?

**Comment by SkabbMask** [comment]
I get two buttons to click on, to chose between two different bridges (I assume, that's what it looks like to me - see first screenshot).
If I click on one, I get (second screenshot)
![Capture](https://user-images.githubusercontent.com/9406559/162004717-98a01989-c5f7-4792-911d-825f4b84a54f.JPG)
![Capture1](https://user-images.githubusercontent.com/9406559/162004889-8067ac2b-2104-4e60-81df-fdf47bcf8ee6.JPG)

**Comment by rootzoll** [comment]
Ah OK your operating diretly on the store - I would not recommend that if you want to use IP2Tor with RaspiBlitz. Only use the SSH menu dialogs to setup IP2Tor bridge first and then add duckdns & letsencrpt. Try to follow this tutorial: https://github.com/rootzoll/raspiblitz#ip2tor-paid

**Comment by SkabbMask** [comment]
I'm not sure what "operating directly on the store" means. I was following this tutorial: https://coincharge.io/en/raspiblitz-btcpay-server/
And from what I can see in the tutorial (ip2tor bridge part at least), it's the same.
To clarify; I did setup the ip2tor bridge through the raspiblitz. I know no other way of doing it 😅

**Comment by rootzoll** [comment]
How do you get the website with the two different bridges to (from the screenshot you posted above). Thats when you use your duckdns domain with https (that you set for IP2Tor)? Do you add the port to the domain? It should be in the end something like this: https://yoursubdomain.duckdns.org:45452

**Comment by SkabbMask** [comment]
Yes. Hmm, I don't remember specifying a port so that might be it then, I'll try and get back. Thanks! :)

**Comment by SkabbMask** [comment]
![Capture](https://user-images.githubusercontent.com/9406559/165604448-22d24b79-6504-4b16-9e58-97785ec76883.JPG)

Well, now I can't even get that far >.<
I've probably fucked it up somehow :(

**Comment by SkabbMask** [comment]
Update: I get to the sign-in page when I add the port to the URL. But it still says that it's unsecure: https://mhm1234567.duckdns.org:47731/

**Comment by SkabbMask** [comment]
It says the cert is issued to "coinpages.duckdns.org"
